org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/IRunnableDeviceService.java

```diff
@@ -205,5 +205,16 @@
      */
 	Collection<DeviceInformation<?>> getDeviceInformationIncludingNonAlive() throws ScanningException;
 
-
+    /**
+     * This is a convenience method for getting the currently active scanner. 
+     * It is useful if the scan is paused and it is required to seek the scan
+     * to a new location from Jython. The returned scanner will be null unless
+     * a scan is currently running, therefore it is of limited usage normally.
+     * 
+     * @return current actively scanning device.
+     */
+	default IRunnableDevice<?> getActiveScanner() {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1452)